### DISCUSSION TITLE
SecOnion setup  for MSSP

### DISCUSSION THREAD

**Discussion by DrStupendous** [Security-Onion-Solutions/securityonion#10558] [Unsupported Versions] [answered]
Hi Everyone,

I was wondering if anyone has SecOnion setup in an MSSP configuration, and how that was done?

What I had in mind is having a forwarder node at each client, and a separate search node for each client tied into a centralized manager node.

The clients would then connect back via to the manager via a site to site VPN?

Would this be possible/ practical or would it work better if the manager was setup in an AWS or AZURE instance?

Thank you in advance

**Comment by InfosecGoon** [comment, marked as answer]
Just a note - it's not really possible to isolate a site's traffic to a particular Search Node like this. All Forwarders send their data back to the Manager, where it goes into a Redis queue, and the Search Nodes then pull from the queue. A zeek or suricata log entry from a particular Forward node could end up on any Search Node in the grid.

If you want to isolate the data like this, you'd probably want to use a Heavy Node instead, but that has some performance penalties when you need to search across them.

**Reply by DrStupendous** [reply]
Thank you very much.

Makes sense as Sec Onion does not support multi tenant deployments.
I am currently trying to work out the best setup I can do for a small client
Taking into consideration available resources in both ram and disk space.
Currently watching one of your videos on how to best tune the NIDS and Winlogbeat to not have a flood of alerts coming in.

I will look into if a heavy node would be a good fit

**Comment by gsprojectacc** [comment]
So that, is it possible to use both Search Node and Heavy Node in single Security Onion distributed deployment?
